### PR TITLE
Try to clarify when to use the issue template

### PR DESCRIPTION
**Pull request by lydell**
I've noticed many people apologizing for not using the issue template where there has been absolutely no reason to apologize! The issue template is not meant for _all_ types of issues.

### CODE REVIEW

**Comment by suchipi** [comment]
@lipis and I with the late night GitHub game, what up 😂

**Comment by lipis** [comment]
hehe.. it's 9:49am here.. go to sleep.. :)

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!-- BUGGY OR UGLY? Please use this template.
+<!-- For ugly or incorrect code issues: Please use this template.
```
Comment (j-f1) [inline .github/ISSUE_TEMPLATE.md]: How about something like

> If your issue isn’t about ugly or incorrect code, feel free to delete this and write whatever you want here.

between lines 1 and 3?

Comment (lydell) [reply .github/ISSUE_TEMPLATE.md]: Might be good. Might also be too long. Not sure.

Comment (lipis) [reply .github/ISSUE_TEMPLATE.md]: Too long.. people tent to ignore long texts.

**Pull request merged**